### PR TITLE
COM-5324: "Subscription Errored" email template.

### PR DESCRIPTION
**Pull request by Kibo-Sagar**

### CODE REVIEW

File: templates/email/subscription-errored.live [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% extends "email/email" %}
```
Comment (kibo-travis-patrick) [inline templates/email/subscription-errored.live]: Why is this a `.live` file when all the other files in this email dir are `.hypr`?

Comment (Kibo-Sagar) [reply templates/email/subscription-errored.live]: `Hypr is the server-side rendering engine. Hypr templates are denoted by the .hypr extension, but Hypr can also render templates with the .hypr.live extension.
HyprLive is the client-side rendering engine. HyprLive templates are denoted by the .hypr.live extension.`

This is the difference, but i guess i've took a reference from "order-confirmation-v2.hypr.live" and mistakenly i have removed .hypr but making it .hypr as most of the template's are .hypr

File: labels/de-DE.json [hunk 2]
```diff
@@ -669,6 +669,7 @@
   "submitForApproval": "Zur Genehmigung einreichen",
   "submittedBy": "Eingereicht von",
   "subtotal": "Zwischensumme",
+  "subscriptionErroredBlob1": "Bonjour, <br/><br/> Der Status Ihres Abonnements Nr. {0} wird auf {1} aktualisiert. Das Abonnement wurde aufgrund eines möglichen Problems mit der Zahlung oder eines anderen Problems in einen fehlerhaften Zustand verschoben. Bitte wenden Sie sich für weitere Einzelheiten an den Kundendienst. Wenn Sie Ihr Abonnement anzeigen oder verwalten möchten, besuchen Sie bitte <a href=\"https://{2}/myaccount\">Mein Konto</a>.",
```
Comment (kibo-travis-patrick) [inline labels/de-DE.json]: Looks like you've used the French hello instead of German. Google translate gives me "Hallo" which I've used.

Comment (Kibo-Sagar) [reply labels/de-DE.json:672]: german is correct, this is german locale file. see the one above "Zwischensumme" means "subtotal" in german.

Comment (Kibo-Sagar) [reply labels/de-DE.json]: fixed it.

**Review by kibo-rajeshdatir — APPROVED** [review]

**Pull request merged**